src/c++/vdo/base/message-stats.c

```diff
@@ -2,8 +2,10 @@
 /*
  * Copyright 2023 Red Hat
  */
+#include <linux/limits.h>
 
```

Comment:
Do you need to include limits.h? It's not clear to me what requires it.

Reply:
left over from when I was trying to use U32_MAX. Will remove.